docs/error_knowledge_base.md

```diff
@@ -463,6 +463,14 @@ def package_info(self):
     self.cpp_info.includedirs = []
 ```
 
+#### **<a name="KB-H072">#KB-H072</a>: "PYLINT EXECUTION"**
+
+Pylint is executed by default over all `conanfile.py` in Conan Center Index and it should not be skipped. It's an important tool which help us keeping a standard level of acceptance, otherwise, it would be incredible hard to review all recipes keeping the same standard level.
```

Comment:
```suggestion
Pylint is executed by default over all `conanfile.py` files in Conan Center Index and it should not be skipped. It's an important tool which helps us keep a standard level of acceptance. Otherwise, it would be incredibly hard to review all recipes and keep them to the same level of standards.
```